NCTRC/frontend/build/test/coverage/block-navigation.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable */
```

Comment:
what's the reason for ignoring our regular linting rules? There doesn't seem to be a good reason for using var here.